Encode.xs

```diff
@@ -139,6 +140,31 @@ do_fallback_cb(pTHX_ UV ch, SV *fallback_cb)
 }
 
 static SV *
+do_bytes_fallback_cb(pTHX_ U8 *s, STRLEN slen, SV *fallback_cb)
+{
+    dSP;
+    int argc;
+    STRLEN i;
+    SV *retval = newSVpvn("",0);
```

Comment:
The allocation of retval here would cause a memory leak if the callback invoked in `call_sv` throws an exception.